### ISSUE TITLE
Robot Raconteur Robot Drivers questions

### ISSUE THREAD

**Issue by wangmunandadajuan** [hehonglu123/robotic_portrait#1]
Hello, I would like to attempt this project in simulation, but I am not very familiar with "Robot Raconteur Robot Drivers" and have been unable to connect my computer to the robot after trying for some time. I have downloaded the files from the "ABB IRC5 [UR cb2]" link (https://github.com/robotraconteur-contrib/URRobotRaconteurDriver), but I am unsure about the command "./ABBRobotRaconteurDriver.exe --robot-info-file=abb_1200_5_90_robot_default_config.yml". Additionally, when running the code snippet:

robot = robot_obj('ABB_1200_5_90', 'G:/Downloads/robotic_portrait-main/config/ABB_1200_5_90_robot_default_config.yml', tool_file_path=CODE_PATH+'config/heh6_pen.csv')
RR_robot_sub = RRN.SubscribeService('rr+tcp://localhost:58651?service=robot')
RR_robot = RR_robot_sub.GetDefaultClientWait(1)

I encounter the error "RobotRaconteur.RobotRaconteurPythonError.ConnectionException: RobotRaconteur.ConnectionError Subscription default client timed out". I'm sorry to bother you, but could you please help me with this issue? Thank you very much.

**Comment by hehonglu123** [comment]
Thanks for your interest! This project is still under heavy development, so the instructions are not as clear. 
This project includes testing on ABB 1200 with IRC5 controller and UR5 with cb2 controller. The Robot Raconteur (RR) drivers to both will be included in binary release this week.
Since you mention simulation, ABB RR drivers support simulation in ABB's RobotStudio, but there's not one for UR. However, RR offers simulation interface with Gazebo in [robotraconteur_training_sim](https://github.com/robotraconteur-contrib/robotraconteur_training_sim).

**Comment by wangmunandadajuan** [comment]
Yes, I am trying to provide a portrait on RobotStudio and then have the robot perform actions, but I cannot achieve this because I am not familiar with the Robot Raconteur driver. I look forward to more detailed explanations from you on this matter. Thank you very much.

**Comment by hehonglu123** [comment]
For RobotStudio simulation, you could use External Guided Motion (EGM) directly to execute the trajectory. The corresponding Python module is [abb_robot_client](https://github.com/rpiRobotics/abb_robot_client).

If you are not familiar with EGM, see [this video](https://youtu.be/vuRuruq5P5k) for EGM setup. The repo in the video is deprecated, so please ignore that repo and only pay attention to the EGM setup. The steps for virtual controller are the same.

Again this project is still under development, feel free to try out but may not be intuitive and have small issues here and there.
